src/components/CheckboxField/__tests__/CheckboxField.spec.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import {
+  expect,
+  test,
+} from '@playwright/experimental-ct-react';
+import {
+  mixPropTests,
+  propTests,
+} from '../../../../tests/playwright';
+import {
+  CheckboxFieldForTest,
+  CheckboxFieldForRefTest,
+  CheckboxWForFormLayoutTests,
+} from './CheckboxField.story';
+import { checkedPropTest } from './_propTests/checkedPropTest';
+
+test.describe('CheckboxField', () => {
+  test.describe('base', () => {
+    test.describe('visual', () => {
+      [
+        ...propTests.labelPropTest,
+        ...propTests.renderAsRequiredPropTest,
+        ...propTests.isLabelVisiblePropTest,
```

Comment:
I would like to have uniform approach so we do not have to thing whether is it is array or function. Lets discuss it first.

Reply:
This comment is outdated as we decided to abandon parametrization.